pandas/formats/format.py

```diff
@@ -1756,13 +1758,501 @@ def __init__(self, row, col, val, style=None, mergestart=None,
                               "vertical": "top"}}
 
 
+class CSSWarning(UserWarning):
+    """This CSS syntax cannot currently be parsed"""
```

Comment:
i would put all of this added code in forma.py in a new module, maybe css.py (or better name)